### DISCUSSION TITLE
Copilot education pack being available

### DISCUSSION THREAD

**Discussion by Kenan7** [community/community#140977] [GitHub Education]
### Hi everyone,

Before, there wasn't any waiting time after being approved, the perks were always there, in a second.

Now that I'm approved, how many days before I get my perks? I just wanted to access Copilot again.

**Comment by davevad93** [comment]
Hi @Kenan7 ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @Kenan7 was approved and your benefits are now available.`
